Assets/Scripts/Editor/ThrowableCheats.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+public class ThrowableCheats : MonoBehaviour
+{
+    [MenuItem(Constants.CHEATS_ADD_MEDICINE_BOTTLES, true)]
+    public static bool ValidateAddMedicineBottles()
+    {
+        return Application.isPlaying && UIManager.Instance;
+    }
+
+    [MenuItem(Constants.CHEATS_ADD_MEDICINE_BOTTLES)]
+    public static void AddMedicineBottles()
+    {
+        GameObject medicineBottlePrefab = Resources.Load<GameObject>("Cheats/Medicine_Bottle_Throwable_CHEATS");
+        
+        for (int i = 0; i < 7; i++)
+        {
+            GameObject spawnedBottle = Instantiate(medicineBottlePrefab);
+            spawnedBottle.transform.Find("Canvas").gameObject.SetActive(false);
```

Comment:
Never really a fan of the Find functions, but for cheats it should be fine!
For OnInteract, don't we hide the canvas afterwards though? I imagine if there's a slight pop-in visually from the Canvas, that should be fine anyways since this is for debugging.

Reply:
Yeah I thought calling OnInteract would hide the Canvas too, but for some reason it doesn't do the trick.  I assumed setting enableInteract = false would work, but the canvas still pops up when I throw.

Reply:
Weird! Maybe @chadleytan has some input on this, but this isn't a big issue anyways!

Reply:
Well right now OnInteract doesn't hide the canvas rn because of the whole refactor of the dialogue. So like even interacting with something still has the interactable (A) show atm. Haven't thought of the best way to hide and then show it when done interacting but definitely plan on tackling that issue when I can.